### ISSUE TITLE
tag is not reported in supercell

### ISSUE THREAD

**Issue by patquem** [LaurentRDC/crystals#9]
Hello,

First of all, many thanks for the 'crystals' library development that is really usefull.
Is there any reason for not copying/pasting 'tag' during supercell creation ?
(in my case, I need 'tag' to be kept during supercell creation)
Thanks.
Patrick

**Comment by LaurentRDC** [comment]
Hi Patrick,

Happy to hear this package is helpful.

There's no reason the `tag` isn't being copied; this appears to be an oversight on my part. The supercell is being [assembled here](https://github.com/LaurentRDC/crystals/blob/master/crystals/crystal.py#L428) and as you can see, the `tag` isn't being properly propagated. Let me fix this quickly

**Comment by patquem** [comment]
Hi @LaurentRDC ,

Thanks for your reply.

Concerning tag, That's what I have seen and consenquently modified the source code to have 'tag' operating.

I have been working with 'crystals' for a few days... and I would have some proposals to ease crystal usage like : 

- make 'supercell' an attribute of Crystal,
- extend real_coords() and frac_coords() to arrays((**N**, 3)), 
- make these 2 last functions (or equivalent)  **Lattice** class functions, ... 

but probably I am not so 'crystal' experienced to use correctly this library yet

**Comment by LaurentRDC** [comment]
Have you taken a look at the user guide? For example [here](https://crystals.readthedocs.io/en/master/guides/crystals.html#supercells), you'll see you can create supercells from `Crystal` objects.

I'm happy to review pull requests that suggest enhancements, but I don't have the time to personally do this. It's also important for me and other researchers that use this package that backwards compatibility be preserved; this means that there are some things that cannot be changed.

Don't hesitate to open a pull request and we'll see what we can do

**Comment by patquem** [comment]
I agree with you and understand your position.
I just find that it's shame that supercell object returned by crystal.supercell() is not a attribute of Crystal.
Consequently, I have created a kind of Crystal 'superclass' to agregate this missing component.

In parallel, I have developped a plot() function to ease crystal 3D visualisation (with 'standard' matplotlib).
Maybe, it could be interesting to pull it in crystals in the future (?)

Regards
![image](https://user-images.githubusercontent.com/16154687/156783966-7d046f37-5430-4b98-89e0-99bcdb8f4b1c.png)
